test/fixtures/test-runner/output/describe_nested.snapshot

```diff
@@ -27,3 +27,18 @@ ok 1 - nested - no tests
 # skipped 0
 # todo 0
 # duration_ms *
+node:assert:173
+  throw err;
+  ^
+
+AssertionError [ERR_ASSERTION]: Unexpected global(s) found: __DT_AGENT_INJECTED__
```

Comment:
I'm not quite sure what this global is, but I don't think it was here before

Reply:
I perhaps did not expect so many commits to line up in the PR. Should I create a fresh PR with the latest changes?
I have changed 3 test_runner/reporter files and 2 snapshots, in total.

Reply:
I can also create 2 different PRs, one for displaying mages in dot router and other for adding a stack trace to it.

Reply:
You could rebase and squash some commits, if this is an issue (it would also allow fixing the first commit message)

Reply:
Thanks!